cblaster/hmm_search.py

```diff
@@ -149,7 +158,10 @@ def run_hmmsearch(fasta, query):
         temp_res: List, String of result file names
     """
     LOG.info("Performing hmmsearch")
-    output = Path(query).with_suffix(".txt")
+    output = Path(cagecat_prefix, query).with_suffix(".txt")

```

Comment:
Here as well, on the CAGECAT side can you just have `cagecat_prefix` as part of `query` when you pass it to the function?